src/api/entities/Identity/Authorizations.ts

```diff
@@ -25,35 +27,51 @@ export class Authorizations extends Namespace<Identity> {
   /**
    * Fetch all pending authorization requests for which this identity is the target
    *
-   * @note supports pagination
+   * @param opts.includeExpired - whether to include expired authorizations. Defaults to true
    */
-  public async getReceived(
-    paginationOpts?: PaginationOptions
-  ): Promise<ResultSet<AuthorizationRequest>> {
+  public async getReceived(opts: {
+    filterByType: AuthorizationType;
```

Comment:
Should just be called `type`, and the docs should say `@param opts.type - fetch only authorizations of this type. Fetches all types if not passed`